### PR TITLE
Fixes #17228: Rudder 6.1 does not start anymore

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/17228

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala [hunk 1]
```diff
@@ -88,6 +88,7 @@ import com.normation.rudder.batch._
 import com.normation.rudder.db.Doobie
 import com.normation.rudder.domain._
 import com.normation.rudder.domain.logger.ApplicationLogger
+import com.normation.rudder.domain.logger.ApplicationLoggerPure
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala:91]: Import not needed i think!

**Comment by fanf** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala (hunk 1 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala]: right!

**Comment by amousset** [comment]
OK, squash merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**